core/src/test/java/org/apache/iceberg/jdbc/TestJdbcCatalog.java

```diff
@@ -630,6 +630,30 @@ public void testCreateNamespaceWithMetadata() {
     Assert.assertTrue(catalog.namespaceExists(testNamespace));
   }
 
+  @Test
+  public void testNamespaceLocation() {
+    Namespace testNamespace = Namespace.of("testDb", "ns1", "ns2");
+
+    // Test with location
+    Map<String, String> testMetadata = ImmutableMap.of();
+    catalog.createNamespace(testNamespace, testMetadata);
+
+    Assert.assertTrue(catalog.loadNamespaceMetadata(testNamespace).containsKey("location"));
```

Comment:
nit: `Assertions.assertThat(catalog.loadNamespaceMetadata(testNamespace)).containsKey("location")` might be preferrable here imo, because it will print the content of the map if the assertion fails